neo/SmartContract/Helper.cs

```diff
@@ -101,11 +101,8 @@ internal static bool VerifyWitnesses(this IVerifiable verifiable, Snapshot snaps
                 byte[] verification = verifiable.Witnesses[i].VerificationScript;
                 if (verification.Length == 0)
                 {
-                    using (ScriptBuilder sb = new ScriptBuilder())
-                    {
-                        sb.EmitAppCall(hashes[i].ToArray());
-                        verification = sb.ToArray();
-                    }
+                    verification = snapshot.Contracts.TryGet(hashes[i])?.Script;
+                    if (verification is null) return false;
```

Comment:
Before was the hash, and now is the script? Contact.Script is not the full script?

Reply:
The old code is: `sb.EmitAppCall(hashes[i].ToArray());`.

Reply:
Ok, so before was a APPCALL to the script, and now is a complete script?

Reply:
Yes. We don't need to discuss too much in this place. Because in the future it will be replaced by SVM (Simplified Verification Model).